test/openssl/test_engine.rb

```diff
@@ -74,8 +74,8 @@ def test_openssl_engine_cipher_rc4
   # this is required because OpenSSL::Engine methods change global state
   def with_openssl(code, **opts)
     assert_separately([{ "OSSL_MDEBUG" => nil }, "-ropenssl"], <<~"end;", **opts)
-      require #{__FILE__.dump}
-      include OpenSSL::TestEngine::Utils
+      require 'engine_utils'
+      include EngineUtils
       #{code}
     end;
   end
```

Comment:
`module Utils` below should be removed

Reply:
I think we should simply inline these helper methods in `Utils`. They are both too specialized and don't look very useful anyway.

Reply:
I've inlined the helpers.